doc/modules/language-guide/pages/motoko-introduction.adoc

```diff
@@ -82,7 +83,7 @@ However, it requires special compiler and type-system support, as we discuss in
 In an _asynchronous_ computing setting, a program and its running environment are permitted to perform _internal computations_ that occur _concurrently_ with one another.
 
 Specifically, asynchronous programs are ones where the program's requests of its environment do not (necessarily) require the program to wait for the environment.
-In the meantime, the program is permitted to make internal progress within this environment while it waits, perhaps _actively_. In the example, above, the program makes the second request while still waiting for the first micro service.
+In the meantime, the program is permitted to make internal progress within this environment while it waits, perhaps _actively_. In the example, above, the program issues the second request before waiting for the first request to complete.
```

Comment:
I don't understand what actively waiting means, but probably not important for this PR..

Reply:
YEah, not my text. maybe, "while the environment proceeds with completing the request."

Reply:
```suggestion
In the meantime, the program is permitted to make internal progress within this environment while the environment proceeds to complete the request. In the example, above, the program issues the second request before waiting for the first request to complete.
```